addons/interaction/functions/fnc_getWeaponAttachmentsActions.sqf

```diff
@@ -0,0 +1,68 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001, Dystopian
+ * Returns children actions for weapon attachment switching.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Actions <ARRAY>
+ *
+ * Example:
+ * player call ace_interaction_fnc_getWeaponAttachmentsActions
+ *
+ * Public: No
+ */
+
+params ["_unit"];
+
+[_unit, {
+    params ["_unit"];
+
+    private _currentWeapon = currentWeapon _unit;
+    if (_currentWeapon isEqualTo "") exitWith {[]};
+    private _weaponItems = _unit weaponAccessories _currentWeapon apply {toLower _x};
+    private _cfgWeapons = configFile >> "CfgWeapons";
+    private _actions = [];
+
+    // "attach" actions
+    private _items = _unit call EFUNC(common,uniqueItems) apply {toLower _x};
+    private _compatibleItems = _currentWeapon call CBA_fnc_compatibleItems apply {toLower _x};
```

Comment:
Can get rid of the `toLower` here and in other places once CBATeam/CBA_A3#1335 is merged. Everything should be in config case.